README.md

```diff
@@ -3,3 +3,5 @@ Dreamstones V2.1
   No limit this time.
 
 ###Sky is limit
+
+And let's talk about this
```

Comment:
je suis pas d'accord. 
car Lorem ipsum dolor sit met, connecteur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna a liqua. Ut enim ad minim venom, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea comido consequat. Luis aute irure dolor in reprehenderit in voluptate relit esse cillim dolore eu fuggita nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit animé id est labour.